src/kuka_sunrise/configuration_manager.cpp

```diff
@@ -30,6 +30,28 @@ ConfigurationManager::ConfigurationManager(
   joint_stiffness_temp_(std::vector<double>(7, 1000.0)),
   joint_damping_temp_(std::vector<double>(7, 0.7))
 {
+  parameter_set_access_rights_.emplace(
+    "command_mode", ParameterSetAccessRights {false, true, true,
+      false, false});
+  parameter_set_access_rights_.emplace(
+    "control_mode", ParameterSetAccessRights {false, true, true,
+      false, false});
+  parameter_set_access_rights_.emplace(
+    "joint_stiffness", ParameterSetAccessRights {false, true,
+      true, false, false});
+  parameter_set_access_rights_.emplace(
+    "joint_damping", ParameterSetAccessRights {false, true, true,
+      false, false});
+  parameter_set_access_rights_.emplace(
+    "send_period_ms", ParameterSetAccessRights {false, true,
+      false, false, false});
+  parameter_set_access_rights_.emplace(
+    "receive_multiplier", ParameterSetAccessRights {false, true,
+      false, false, false});
+  parameter_set_access_rights_.emplace(
+    "controller_ip", ParameterSetAccessRights {false, false,
+      false, false, true});
```

Comment:
Does the program stand in configuring state long enough for you to be able to set the ip?

Reply:
Answer: yes, from cmd line with ros2 node run cmd or from launch file.